Python/sysmodule.c

```diff
@@ -3358,6 +3358,16 @@ make_impl_info(PyObject *version_info)
     if (res < 0)
         goto error;
 
+#ifdef VARCH_NAME
+    value = PyUnicode_FromString(VARCH_NAME);
+    if (value == NULL)
+        goto error;
+    res = PyDict_SetItemString(impl_info, "arch", value);
+    Py_DECREF(value);
+    if (res < 0)
+        goto error;
```

Comment:
For new code, please use braces around if:

```suggestion
    if (value == NULL) {
        goto error;
    }
    res = PyDict_SetItemString(impl_info, "arch", value);
    Py_DECREF(value);
    if (res < 0) {
        goto error;
    }
```

Reply:
I have made changes, Please reviewer changes.